jenkins/e2e-image/Dockerfile

```diff
@@ -51,26 +51,38 @@ RUN apt-get update && \
 
 RUN cd /tmp && \
     git clone https://github.com/google/jsonnet.git && \
-    ( cd jsonnet && \
-      make jsonnet && \
-      cp jsonnet /bin \
-    ) && \
-    rm -rf /tmp/jsonnet
-
-RUN mkdir -p /tmp/terraform/ && \
-    ( cd /tmp/terraform && \
-      wget https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip && \
-      unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip -d /bin \
-    ) && \
+    cd jsonnet && \
+    make jsonnet && \
+    cp jsonnet /bin && \
+    rm -rf /tmp/jsonnet && \
+    mkdir -p /tmp/terraform/ && \
+    cd /tmp/terraform && \
+    wget https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip && \
+    unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip -d /bin && \
     rm -rf /tmp/terraform
 
+ADD ["https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.tar.gz", \
```

Comment:
https://docs.docker.com/engine/reference/builder/#/add

`The ADD instruction copies new files, directories or remote file URLs from <src> and adds them to the filesystem of the image at the path <dest>.`